src/test/java/com/google/sps/servlets/NewClassTest.java

```diff
@@ -56,15 +59,33 @@ public void tearDown() {
 
```

Comment:
You might want to add a test case that modifies an existing user and verifies that the user was modified correctly (e.g. ta already exists and is registered to a class and owns a class now verify it's still registered to that class and owns both the old and new classes)

Reply:
Done